lighthouse/routes/v3/beckman_v3_routes.py

```diff
@@ -0,0 +1,48 @@
+from flask import Blueprint, request
+from flask_cors import CORS
+
+from lighthouse.types import FlaskResponse
+
+from lighthouse.constants.fields import FIELD_EVENT_TYPE
+from lighthouse.classes.beckman_v3 import Beckman
+
+from lighthouse.hooks.beckman_events import create_plate_event
+from lighthouse.routes.common.beckman import get_failure_types, get_robots
+from lighthouse.routes.common.cherrypicked_plates import create_plate_from_barcode, fail_plate_from_barcode
+from lighthouse.routes.common.plate_events import create_plate_event as v1_create_plate_event
```

Comment:
If you rename this as the `v1_create_plate_event` it probably shouldn't be defined in `common`.  I would instead import the v1 method from the v1 blueprint file and not from common.